client/my-sites/customer-home/cards/features/quick-start/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { Button, Card } from '@automattic/components';
+import { connect } from 'react-redux';
+import 'moment-timezone';
+
+/**
+ * Internal dependencies
+ */
+import HappinessEngineersTray from 'components/happiness-engineers-tray';
+import CardHeading from 'components/card-heading';
+import {
+	withAnalytics,
+	composeAnalytics,
+	recordTracksEvent,
+	bumpStat,
+} from 'state/analytics/actions';
+import { navigate } from 'state/ui/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import QueryConciergeInitial from 'components/data/query-concierge-initial';
+import getConciergeNextAppointment from 'state/selectors/get-concierge-next-appointment';
+import { withLocalizedMoment } from 'components/localized-moment';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const QuickStart = ( {
+	bookASession,
+	moment,
+	nextSession,
+	reschedule,
+	siteId,
+	siteSlug,
+	translate,
+	viewDetails,
+} ) => {
+	return nextSession ? (
+		<Card className="quick-start next-session">
+			<HappinessEngineersTray />
+			<CardHeading>{ translate( 'Your scheduled Quick Start support session' ) }</CardHeading>
+			<div className="quick-start__date">
+				{ translate( 'Date' ) }
+				{ moment( nextSession.beginTimestamp ).format( 'llll' ) }
+			</div>
+			<div className="quick-start__time">
+				{ translate( 'Time' ) }
+				{ moment.tz( nextSession.beginTimestamp, moment.tz.guess() ).format( 'LT z' ) }
+			</div>
+			<Button onClick={ () => viewDetails( siteSlug ) }>{ translate( 'View details' ) }</Button>
+			<a
+				href={ `/me/concierge/${ siteSlug }/book` }
+				onClick={ () => reschedule( siteSlug, nextSession.id ) }
+			>
+				{ translate( 'Reschedule' ) }
+			</a>
+		</Card>
+	) : (
+		<>
+			{ siteId && <QueryConciergeInitial siteId={ siteId } /> }
+			<Card className="quick-start book-session">
+				<HappinessEngineersTray />
+				<CardHeading>{ translate( 'Schedule time with an expert' ) }</CardHeading>
+				<p>
+					{ translate(
+						'Need help with your site? Set up a video call to get hands-on 1-on-1 support.'
+					) }
+				</p>
+				<Button onClick={ () => bookASession( siteSlug ) }>
+					{ translate( 'Book a session' ) }
+				</Button>
+			</Card>
+		</>
+	);
+};
+
+export default connect(
+	( state ) => ( {
+		siteId: getSelectedSiteId( state ),
+		siteSlug: getSelectedSiteSlug( state ),
+		nextSession: getConciergeNextAppointment( state ),
```

Comment:
This needs to also compare blog ID now (it should only be `true` if there is a session, _and_ it's for the current blog being viewed.

Reply:
Hm, there _is_ no blog ID associated with the appointment object. I'll have to dig into how the concierge pages themselves handle similar checks.

Reply:
Don't need to worry about this anymore.